heareval/tasks/dcase2016_task2.py

```diff
@@ -88,7 +87,9 @@ def get_split_metadata(self, split: str) -> pd.DataFrame:
                 .replace("annotation", "sound")
                 .replace(".txt", ".wav")
             )
-            assert os.path.exists(sound_file)
+            # Remove the assert statement as this file might not exist.
+            # This is anyways ensured in the Base ExtractMetadata task.
+            # assert os.path.exists(sound_file)
```

Comment:
Removed this `assert` because, in the case of sampled data, files in the metadata might not exist in the folder. This check has been moved in the `run` part of the `ExtractMetadata` task. If files are not found, the count of files not found will be printed and those rows will be removed from the metadata so that there is no error while subsampling from the small dataset.